core/App/DefaultConfiguration.ts

```diff
@@ -0,0 +1,15 @@
+import HomeContentView from './components/views/HomeContentView'
```

Comment:
My only comment is that we use camelCase for .ts files. Unlike components (.tsx files) in which case we use PascalCase.

Reply:
@amanji , can that be codified as a linter rule?

Reply:
Yeah we can do that